src/elf/reloc.rs

```diff
@@ -132,7 +132,7 @@ macro_rules! elf_rela_std_impl { ($size:ident, $isize:ty) => {
             impl From<Rel> for Reloc {
                 fn from(rel: Rel) -> Self {
                     Reloc {
-                        r_offset: rel.r_offset as usize,
+                        r_offset: rel.r_offset as u64,
                         r_addend: 0,
```

Comment:
We may also want this to be an `Option`, but I didn't do it cause i was being lazy ;)

Reply:
Since this is a breaking change anyways, I turned it into an `Option` as part of this PR.